Monika After Story/game/script-greetings.rpy

```diff
@@ -371,3 +371,52 @@ label monikaroom_greeting_post:
         set_keymaps()
     return
 
+label_greeting_youarereal:
+default persistent.you = True
+    python:
+        try: 
+            renpy.file("/characters/" + persistent.playername + ".chr")
+            persistent.you = True
+        except IOError:
+            persistent.you = False
+    m 1b "[player]! Great to see you!"
+
+if persistent.you:
+    jump youreal
+else:
+    return
```

Comment:
Because greetings are chosen randomly and only one is added per game a launch, you should add some sort of actual greeting dialogue even if `player.chr` doesnt exist. It could be something where monika talks about `chr` files or some shit, but the main thing is she needs to say something. You can't have a slient greeting flow.